src/OSPSuite.Presentation/DTO/ObjectBaseDTO.cs

```diff
@@ -42,35 +44,20 @@ protected virtual bool IsNameUnique(string newName)
 
       protected bool IsDescriptionDefined(string description)
       {
-         if (!DescriptionRequired)
-            return true;
-
-         return description.StringIsNotEmpty();
+         return !DescriptionRequired || description.StringIsNotEmpty();
       }
 
       private static class ObjectBaseDTORules
       {
-         public static IBusinessRule UniqueName
-         {
-            get
```

Comment:
Create once per object before. Now created once only! this should help especially with the core ObectDTO!